src/debugpy/_vendored/pydevd/_pydev_bundle/pydev_console_utils.py

```diff
@@ -447,6 +448,7 @@ def getFrame(self):
 
         return xml.getvalue()
 
+    @silence_warnings_decorator
```

Comment:
Note that I'm silencing in the caller and not in the actual resolver to avoid having to start/stop to catch the warnings for each new variable.